cmd/proxy/main.go

```diff
@@ -203,10 +204,11 @@ func exit(log *zap.SugaredLogger, err error, wrapStrs ...string) {
 
 func readinessTCPHandler(port int, logger *zap.SugaredLogger) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
-		timeout := time.Duration(1) * time.Second
-		address := net.JoinHostPort("localhost", strconv.FormatInt(int64(port), 10))
+		ctx := r.Context()
```

Comment:
Shouldn't we use the request's context instead?

Reply:
Oh it's right there, lol.

Reply:
that's exactly what this is

Reply:
Yep. Thought an empty context was created.